src/index.ts

```diff
@@ -55,7 +56,7 @@ async function clientNewAccount(r: NginxHTTPRequest): Promise<void> {
   } catch (e) {
     const errMsg = `Error creating ACME account. Error=${e}`
     log.error(errMsg)
-    return r.return(500, errMsg)
+    return null; //r.return(500, errMsg)
```

Comment:
what happens if a `js_periodic` handler throws an exception?

Reply:
I'll test it.

Reply:
I added this line near the top of `clientAutoMode()`:
```
throw 'oopsies'
```

NGINX started up fine, but after a minute (the js_periodic period), this was output in the logs:
```
njs-acme-nginx-1   | 2023/09/08 21:57:56 [error] 26#26: js exception: Error: unhandled promise rejection: oopsies
```